src/pyshark/capture/live_capture.py

```diff
@@ -55,6 +55,16 @@ def __init__(self, interface=None, bpf_filter=None, display_filter=None, only_su
             self.interfaces = [interface]
         else:
             self.interfaces = interface
+        self.check_valid_interfaces(tshark_path=tshark_path)
+        
+    def check_valid_interfaces(self, tshark_path=None):
```

Comment:
I think ti's better to do this when attempting to capture rather than in the constructor, as this may cause unexpected behaviour